lib/pharos/scripts/wait-kubelet-proxy.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 echo "Waiting kubelet-proxy to launch on port 6443..."
 
-while ! (echo > /dev/tcp/localhost/6443) >/dev/null 2>&1; do
+while ! ( bash -c 'echo > /dev/tcp/localhost/6443') >/dev/null 2>&1; do
```

Comment:
Fixes regression from CentOS/RHEL feature.. needs to call explicitly bash because Ubuntu has dash as sh.